.gitignore

```diff
@@ -1 +1,2 @@
 .env
```

Comment:
Rewrite this to `*.env*`

Reply:
alright

Reply:
i have  uploaded the requirements.txt file do we need that?